SECURITY.md

```diff
@@ -6,13 +6,22 @@ If you believe you've found an exploitable security issue in arazzo-engine,
 ## Supported versions
 
 ### Arazzo Runner
-This is the list of versions of `arazzo-runner` which are
+This is the list of versions of `arazzo_runner` which are
```

Comment:
Just as side not for future ref, there seem to be a lacking consistency in naming/commands/packages/docs, with multiple versions being used concurently:

* runner - arazzo_runner - arazzo-runner
* generator - arazzo_generator - arazzo-generator